src/xdocs/checks.xml

```diff
@@ -805,5 +799,26 @@
       </tr>
     </table>
     </section>
+
+    <section name="Non-Standard Checks">
+    <p>
+      These checks aren't normal checks and are usually associated with a specialized filter
+      for information the filter can't get on it's own.
+    </p>
+
+    <table>
+      <tr>
+        <td><a href="config_misc.html#FileContentsHolder">FileContentsHolder</a></td>
+        <td>Holds the current file contents for global access when configured as a
+        TreeWalker sub-module.</td>
+      </tr>
+      <tr>
+        <td><a href="config_annotation.html#SuppressWarningsHolder">SuppressWarningsHolder</a></td>
+        <td>
+          This check allows for finding code that should not be reported by Checkstyle
```

Comment:
We mislead user ... .
"This check"  --> "This module" . or do description as it is in FileContentsHolder "Holds the .........".
For this short description I usually grap first line from module description.